atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -465,6 +487,10 @@ def run(self):
 
         koji_metadata, output_files = self.combine_metadata_fragments()
 
+        if is_scratch_build():
+            self.upload_scratch_metadata(koji_metadata, server_dir, self.session)
+            return
```

Comment:
Is this `return` ok here? (I don't think so)

Reply:
this plugin should do just 1 thing for scratch build and that is to upload that metadata file, so after that we are done for scratch

Reply:
How about that `build_info` later? Don't we want to still log it?

Reply:
that is again for build and output from cgimport, but we are doing scratch and we don't have build neither are importing anything

Reply:
ok